### PR TITLE
Allowing pulling a container list of more than 15 containers

### PR DESCRIPTION
**Pull request by ami-fairfly**
We could also make a loop that keeps pulling the next page but I wanted a quick fix :)

### CODE REVIEW

**Comment by ericclemmons** [comment]
@ami-fairfly I didn't realize, is 15 the default?

**Comment by ami-fairfly** [comment]
Oops, its 25. Changed it now.

**Comment by ami-fairfly** [comment]
It's undocumented but we have more than 25 containers so it was easy to observe :)

File: lib/reload-nginx.js [hunk 1]
```diff
@@ -20,7 +21,7 @@ try {
 */
 export default function() {
   // list all containers
-  dockerCloud('/api/app/v1/container/')
+  dockerCloud('/api/app/v1/container/?limit=' + containerLimit)
```
Comment (willrstern) [inline lib/reload-nginx.js]: Let's make this an ES6 template string just in case we ever add more params.

**Comment by willrstern** [comment]
great catch @ami-fairfly.  We're just now running 28 as of Friday, so I can confirm this as a problem/fix.

:+1: with one tiny change.

File: lib/reload-nginx.js (hunk 1 above)
Comment (ami-fairfly) [inline lib/reload-nginx.js]: Good idea!

**Comment by ami-fairfly** [comment]
@willrstern There we go

**Comment by willrstern** [comment]
:fire:

**Comment by willrstern** [comment]
released as `willrstern/docker-cloud-nginx:1.3`

**Pull request merged**